cockatrice/src/dlg_settings.cpp

```diff
@@ -868,15 +867,12 @@ MessagesSettingsPage::MessagesSettingsPage()
 
     aAdd = new QAction(this);
     aAdd->setIcon(QPixmap("theme:icons/increment"));
-    aAdd->setToolTip(tr("Add New Message"));
     connect(aAdd, SIGNAL(triggered()), this, SLOT(actAdd()));
     aEdit = new QAction(this);
     aEdit->setIcon(QPixmap("theme:icons/pencil"));
-    aEdit->setToolTip(tr("Edit Message"));
     connect(aEdit, SIGNAL(triggered()), this, SLOT(actEdit()));
     aRemove = new QAction(this);
     aRemove->setIcon(QPixmap("theme:icons/decrement"));
-    aRemove->setToolTip(tr("Remove Message"));
     connect(aRemove, SIGNAL(triggered()), this, SLOT(actRemove()));
```

Comment:
Hmm. Can you briefly explain why you move those?
The other tooltips for the add/remove/edit URL buttons of the card sources are placed similarly in the same file.

Reply:
I move these to the retranslate function because otherwise they will not be changed when you change the language setting

Reply:
Oh, great catch.

I recently added missing tooltips to the custom picture URL buttons in settings as well. The same issue does probably apply there as well. Can you have a look, please?
https://github.com/ebbit1q/Cockatrice/blob/4531_color_picker/cockatrice/src/dlg_settings.cpp#L568-L576

Reply:
yes, these are in the wrong spot, in fact anything that has the tr() function in it should be in the retranslate function

Reply:
Understandable, it takes focus away.
Probably good practice to just open issues to keep track for later and raise awareness. 👍